spec/features/idv/analytics_spec.rb

```diff
@@ -39,7 +39,6 @@
       'IdV: final resolution' => { success: true },
       'IdV: personal key visited' => {},
       'IdV: personal key submitted' => {},
-      'Frontend: IdV: show personal key modal' => {},
```

Comment:
I think it'd be a good idea to add the new event we're expecting in these lists of expected events.

Reply:
Added.